x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_routing/current_stream_entry.tsx

```diff
@@ -32,14 +40,36 @@ export function CurrentStreamEntry({ definition }: { definition: WiredStreamGetR
     }
   );
 
+  const { euiTheme } = useEuiTheme();
+
   return (
     <>
       {!isRoot(definition.stream.name) && (
-        <EuiBreadcrumbs breadcrumbs={breadcrumbs} truncate={false} />
+        <EuiBreadcrumbs
+          max={2}
+          breadcrumbs={breadcrumbs}
+          truncate={false}
+          css={css`
+            padding-bottom: 8px;
```

Comment:
Don't use arbitrary values, please get them from the team such `padding-bottom: ${euiTheme.size.s}`